messenger/Pages/ConversationsPage.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { Component, Fragment } from "react";
+import { View, Text } from "react-native"
+import { useSelector } from "react-redux";
+import global_styles from "../globals";
+
+const ConversationsPage = () => {
+
```

Comment:
You tend to have newlines after the leading `{` but no newline before the trailing `}`. I'm in favor of deleting this newline.